agent/src/cli/cody-bench/strategy-chat.ts

```diff
@@ -57,12 +61,16 @@ export async function evaluateChatStrategy(
         if (response.type === 'transcript') {
             const reply = response.messages.at(-1)
             if (reply?.text) {
-                console.log({ reply })
+                const response = PromptString.unsafe_fromLLMResponse(reply.text)
```

Comment:
Should probably be `unsafe_fromUserQuery`
```suggestion
                const response = PromptString.unsafe_fromUserQuery(reply.text)
```

Reply:
this is the reply from the LLM though!

Reply:
Aaah, my bad!